demo_packet.go

```diff
@@ -62,7 +62,7 @@ func (ms pendingMessages) Less(i, j int) bool {
 func (p *Parser) onCDemoPacket(m *dota.CDemoPacket) error {
 	// Create a slice to store pending mesages. Messages are read first as
 	// pending messages then sorted before dispatch.
-	ms := make(pendingMessages, 0)
+	ms := make(pendingMessages, 0, 10)
```

Comment:
this pattern, `make(T, 0)` appears in a lot of places. could probably find more of them.